### ISSUE TITLE
Syntax highlighting for tags break with dashes in name

### ISSUE THREAD

**Issue by baudren** [baudren/NoteOrganiser#58]
A tag line like this:

```
# something, something else
```

gets properly highlighted in red. However,

```
# something-bad, something-good
```

does not appear in red. Back to regex!

**Comment by baudren** [comment]
b278526 fixes this. I am trying also to highlight titles now.

**Comment by baudren** [comment]
This has been merged now with #66, and the new issue for titles is #67
